lxd/instance/instance_utils.go

```diff
@@ -1202,10 +1202,18 @@ func MoveTemporaryName(inst Instance) string {
 	return fmt.Sprintf("lxd-move-of-%s", inst.LocalConfig()["volatile.uuid"])
 }
 
-// IsSameLocgicalInstance returns true if the supplied Instance and db.Instance have the same project, and either
-// the same name or volatile.uuid values.
+// IsSameLocgicalInstance returns true if the supplied Instance and db.Instance have the same project and name or
+// if they have the same volatile.uuid values.
 func IsSameLocgicalInstance(inst Instance, dbInst *db.Instance) bool {
```

Comment:
Typo in the function name, should be `IsSameLogicalInstance`?

Reply:
Good spot, fixed!